### PR TITLE
Fix Datalake tests

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by xiafu-msft** [comment]
It looks like the recoded file is `? !!python/object/new:multidict._istr.istr` but I remember it should be `? !!python/object/new:multidict._multidict.istr`?

**Comment by rakshith91** [comment]
> It looks like the recoded file is `? !!python/object/new:multidict._istr.istr` but I remember it should be `? !!python/object/new:multidict._multidict.istr`?

good catch..had a previous version of multidict..updated it.

File: sdk/storage/azure-storage-file-datalake/tests/test_directory.py [hunk 1]
```diff
@@ -314,12 +314,11 @@ def test_rename_from_a_directory_in_another_file_system(self):
         properties = new_directory_client.get_directory_properties()
 
         self.assertIsNotNone(properties)
-        self.dsc.delete_file_system(old_file_system_name)
```
Comment (mikeharder) [inline sdk/storage/azure-storage-file-datalake/tests/test_directory.py]: Why is the delete unnecessary?

Comment (rakshith91) [reply sdk/storage/azure-storage-file-datalake/tests/test_directory.py]: teardown already takes care of it.

**Pull request merged**